### ISSUE TITLE
GT++ and DircordSRV Plugin wont't work together (okio.class)

### ISSUE THREAD

**Issue by Dream-Master** [draknyte1/GTplusplus#178]
The newest gt++ version 1.6.110 have some class with the same name then the Discord Bot we using on the server (GTNH) okio.class

https://cdn.discordapp.com/attachments/222778766592573442/395155679603720194/unknown.png

we using DiscordSRV as plugin and now both bots are down

can you say somethings about it ? 

Discord bot give me this error on load:

```
[18:57:17] [DiscordSRV - Initialization/INFO]: [DiscordSRV] DiscordSRV is up-to-date. (de213ef335b25df6d667b19e7d339cd5b6f4d0bc)
[18:57:18] [DiscordSRV - Initialization/INFO]: [DiscordSRV] Language initialized as English
[18:57:18] [DiscordSRV - Initialization/INFO]: [DiscordSRV] Thank you so much to these people for allowing DiscordSRV to grow to what it is: Ac3dUd3, Androkai, anton8604, buildblox, Bysokar, CoreyD97, DeeKay, Diamondman121314/DMAN, FataL, Foxy, geekhideout, Helmic, KaizenWorlds, LDSG Ghost, Mick, Mitecraft, Rui, Ryan (The Loyal One), Sicnus
[18:57:19] [DiscordSRV - Initialization/ERROR]: [DiscordSRV] An unknown error occurred building JDA...
[18:57:19] [DiscordSRV - Initialization/WARN]: java.lang.RuntimeException: java.lang.NoSuchMethodError: okio.BufferedSource.readHexadecimalUnsignedLong()J
[18:57:19] [DiscordSRV - Initialization/WARN]:     at github.scarsz.discordsrv.dependencies.jda.core.requests.RestAction.complete(RestAction.java:376)
[18:57:19] [DiscordSRV - Initialization/WARN]:     at github.scarsz.discordsrv.dependencies.jda.core.entities.impl.JDAImpl.verifyToken(JDAImpl.java:185)
[18:57:19] [DiscordSRV - Initialization/WARN]:     at github.scarsz.discordsrv.dependencies.jda.core.entities.impl.JDAImpl.login(JDAImpl.java:132)
[18:57:19] [DiscordSRV - Initialization/WARN]:     at github.scarsz.discordsrv.dependencies.jda.core.JDABuilder.buildAsync(JDABuilder.java:504)
[18:57:19] [DiscordSRV - Initialization/WARN]:     at github.scarsz.discordsrv.DiscordSRV.init(DiscordSRV.java:324)
[18:57:19] [DiscordSRV - Initialization/WARN]:     at java.lang.Thread.run(Thread.java:745)
[18:57:19] [DiscordSRV - Initialization/WARN]: Caused by: java.lang.NoSuchMethodError: okio.BufferedSource.readHexadecimalUnsignedLong()J
[18:57:19] [DiscordSRV - Initialization/WARN]:     at github.scarsz.discordsrv.dependancies.okhttp3.internal.http1.Http1Codec$ChunkedSource.readChunkSize(Http1Codec.java:444)
[18:57:19] [DiscordSRV - Initialization/WARN]:     at github.scarsz.discordsrv.dependancies.okhttp3.internal.http1.Http1Codec$ChunkedSource.read(Http1Codec.java:425)
[18:57:19] [DiscordSRV - Initialization/WARN]:     at github.scarsz.discordsrv.dependancies.okhttp3.internal.Util.skipAll(Util.java:171)
[18:57:19] [DiscordSRV - Initialization/WARN]:     at github.scarsz.discordsrv.dependancies.okhttp3.internal.Util.discard(Util.java:153)
[18:57:19] [DiscordSRV - Initialization/WARN]:     at github.scarsz.discordsrv.dependancies.okhttp3.internal.http1.Http1Codec$ChunkedSource.close(Http1Code
https://github.com/Scarsz/DiscordSRV/issues/285
GitHub
java.lang.NoSuchMethodError: okio.BufferedSource.readHexadecimalUnsignedLong · Issue #285 · Scarsz/DiscordSRV
```
https://github.com/Scarsz/DiscordSRV/issues/285

**Comment by Namikon** [comment]
I might be of help here:
It appears that the included okio library is incomplete/stripped down or is outdated/from a different feature branch.
So please, if you could either include the full library okio as it exists on Github, or, if you have some special modifications to it, change the namespace so it doesn't conflict with other included okio instances. It would be highly appreciated!

That would fix our, and probably other problems in the future too.

Thanks in advance!
-Namikon

**Comment by draknyte1** [comment]
I'll see if I can update my okio dependency. Might not be resolvable though.

**Comment by LameGirlyCurly** [comment]
Still same issue with 1.7.0-prerelease-8-final. And it's with all available minecraft-discord mods/plugins that one can find :(

**Comment by draknyte1** [comment]
Two things:
1.7.0-8 isn't the latest version. 
It was fixed after 1.7.0-8.
